exercises/concept/parsing-log-files/parsing_log_files.go

```diff
@@ -0,0 +1,21 @@
+package parsing_log_files
+
+func IsValidLine(text string) bool {
+	panic("IsValidLine not implemented")
```

Comment:
```suggestion
	panic("Please implement the IsValidLine function")
```

To be consistent with the majority of exercises in the track. Can you also update the other stubs on this file to follow this format?